api/app/Http/Controllers/UserController.php

```diff
@@ -1,9 +1,34 @@
 <?php namespace App\Http\Controllers;
 
+use Illuminate\Http\Request;
+use App\Repositories\UserRepository as Repository;
+use App\Http\Validators\UserValidator as Validator;
 
 class UserController extends BaseController
 {
+    /**
+     * Assign dependencies.
+     *
+     * @param  Repository  $repository
+     * @param  Validator   $validator
+     * @return void
+     */
+    public function __construct(Repository $repository, Validator $validator)
+    {
+        $this->repository = $repository;
+        $this->validator = $validator;
+    }
 
-    public static $model = 'App\Models\User';
-
+    /**
+     * Put an entity.
+     *
+     * @param  string   $id
+     * @param  Request  $request
+     * @return Response
+     */
+    public function putOne($id, Request $request)
+    {
+        parent::putOne($id, $request);
+        return response(null, 204);
```

Comment:
Ah sorry I think I must have made a mistake with the initial controller requirements - the api should really only return the entity if it has actually changed, so the BaseController should be returning 204; the `User` entity is not a special case.